src/DynamoCore/PackageManager/PackageLoader.cs

```diff
@@ -17,6 +17,7 @@ public struct LoadPackageParams
         public string Context { get; set; }
```

Comment:
`PackageLoader` would need an instance of `IPathManager` to help with its path resolution.